### ISSUE TITLE
ap get higher when validation loss added

### ISSUE THREAD

**Issue by Divenirelrn** [jwyang/faster-rcnn.pytorch#868]
I modify the raw code of pytorch v1.0 version to add validation loss computing capacity, here are my modifications:
first, i add ground truth labels in roibatchLoader:
![image](https://user-images.githubusercontent.com/53744518/127654558-d6cf9bd6-2da4-4fd9-9cf7-5eb0c6febef3.png)
then, i replace if self.training with if True in two files to compute both train and  validation loss:
one in faster_rcnn.py:
![image](https://user-images.githubusercontent.com/53744518/127656866-65837178-6f82-4d03-982c-368ad226b5f9.png)
![image](https://user-images.githubusercontent.com/53744518/127656901-e15cbbaf-5f6f-400d-83bf-d5016b1b80f7.png)
another in prn.py:
![image](https://user-images.githubusercontent.com/53744518/127657027-8e410a46-39ed-49d3-822c-82b4ebb612b6.png)

When above modifications are made, i test the model trained with vgg16 and i get map of about 70% which is higher than origin inal about 65%, so here is my question, why does this happen?
